src/codewhisperer/service/transformByQHandler.ts

```diff
@@ -114,18 +114,36 @@ export async function validateProjectSelection(project: vscode.QuickPickItem) {
     const baseCommand = 'javap'
     const args = ['-v', classFilePath]
     const spawnResult = spawnSync(baseCommand, args, { shell: false, encoding: 'utf-8' })
```

Comment:
For capturing output and other non-trivial handling, did you consider using `ChildProcess` https://github.com/aws/aws-toolkit-vscode/blob/961ff52e05ed45d25c749a2237de6b2b3237f076/src/shared/utilities/childProcess.ts#L68

Reply:
I have not previously considered using `ChildProcess`, added a `TO-DO` comment for this. There will very likely be some changes to the 4 `spawnSync` calls that we have in the next 1-2 weeks (may remove a couple of them, or at least modify them), so I'd like to wait until we finalize that and reserve this consideration for another PR

Reply:
+1 To look into this refactor, but out of scope here. @dhasani23 Make sure to capture this to review internally.